cli/uninstall/uninstall_test.go

```diff
@@ -145,7 +145,7 @@ func TestSearchLatestVersion(t *testing.T) {
 	}
 }
 
-func TestGetAllVersionFormats(t *testing.T) {
+func Test_getAllVersionFormats(t *testing.T) {
```

Comment:
If you make such changes, then apply them at least for the entire file, and even better for the package.
```suggestion
func TestGetAllVersionFormats(t *testing.T) {
```